lib/gemfury/command/app.rb

```diff
@@ -240,7 +240,7 @@ def git_config(repo)
     end
   end
 
-  desc "git:config:set", "Update Git repository's build environment"
+  desc "git:config:set NAME VARS", "Update Git repository's build environment. VARS: KEY1=VAL1 KEY2=VAL2"
```

Comment:
I think for simplicity, we can do a single variable syntax with ellipsis to imply more. 

```suggestion
  desc "git:config:set REPO_NAME KEY=VALUE...", "Update Git repository's build environment."
```